<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewPatient</class>
 <widget class="QMainWindow" name="ViewPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 240);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>List of doctors</string>
     </attribute>
     <widget class="QTableView" name="tableView_doctors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>661</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Experience(years)</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_from">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>82</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_to">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>82</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>from</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_find">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>90</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <property name="text">
       <string>find</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>speciality type:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_getPop">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>200</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <property name="text">
       <string>Get most
popular doctor</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selectDoc">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>280</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <property name="text">
       <string>Select doctor</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>My appoitments</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter email:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(226, 226, 226);</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>901</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableView" name="tableView_patAppointments">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>511</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(226, 226, 226);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_info">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>40</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(226, 226, 226);</string>
       </property>
       <property name="text">
        <string>get info about
 appointments
 on particular day</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Your appointments:</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_calendar">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>110</y>
         <width>371</width>
         <height>431</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QCalendarWidget" name="calendarWidget_certainDate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>341</width>
          <height>201</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(226, 226, 226);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_getinfo">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(226, 226, 226);</string>
        </property>
        <property name="text">
         <string>get info</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>40</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(226, 226, 226);</string>
       </property>
       <property name="text">
        <string>Cancel appointment</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>933</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
